src/models/sunburst.js

```diff
@@ -185,6 +186,7 @@ nv.models.sunburst = function() {
         height:     {get: function(){return height;}, set: function(_){height=_;}},
         mode:       {get: function(){return mode;}, set: function(_){mode=_;}},
         id:         {get: function(){return id;}, set: function(_){id=_;}},
+        container:  {get: function(){return container;}, set: function(_){container=_;}},
```

Comment:
Yeah, it should be read-only, my bad. I just have to set the get function then and remove the set part?